kubesae/utils.py

```diff
@@ -145,7 +145,42 @@ def count_backups(c, bucket_identifier='caktus-hosting-services-backups', profil
         print(f"{v:03d}: {k}\n")
 
 
+@invoke.task
+def scale_app(c, down=False, celery=False):
+    """ A utility to simplify scaling namespace app pods and optionally celery pods.
+
+    Developed primarily to assist with backup verifications. Following the instructions in
+    https://caktus.github.io/developer-documentation/reference/backups/
+
+    If the namespace is using a standard django-k8s celery deployment, you can specify --celery in the command to
+    scale the celery worker and beat appropriately.
+
+    Default behavior is non-destructive, if you accidentally call this task it will merely be a no-op on a
+    running stack. However, if you wish to scale the app down to zero so the database can be dropped the usage is:
+
+    Usage:
+        inv staging project.scale-app --down
```

Comment:
When I ran this command, it complained `No idea what 'project.scale-app' is!`. Should `project` be replaced with `utils` here, and in the following 2 lines?

Reply:
derp